scheds/c/scx_central.c

```diff
@@ -60,14 +61,23 @@ int main(int argc, char **argv)
 	skel->rodata->nr_cpu_ids = libbpf_num_possible_cpus();
```

Comment:
Not from this PR but as a side note, this is wrong, strictly speaking. `nr_cpu_ids` should be `MAX_CPU_ID + 1` while `libbpf_num_possible_cpus()` returns the number of all possible CPUs. The two agree on most systems but if possible cpumask can have holes, the two deviate and there are systems with holes. So, strictly speaking, we should be getting nr_cpu_ids from topology information and then testing the cpus against possible mask.